test/installer/scripts/test_omsadmin.sh

```diff
@@ -1,42 +0,0 @@
-BASE_DIR=$1
-RUBY_TESTVERS=$2
```

Comment:
If that is the case, please update the PR description to state "duplicate" instead of "redundant".